### DISCUSSION TITLE
Duration parameter int to float or minutes to seconds

### DISCUSSION THREAD

**Discussion by do-me** [reinterpretcat/vrp#120] [General]
I just noted that e.g. in the [csv conversion script](https://reinterpretcat.github.io/vrp/getting-started/import.html?highlight=minutes#jobs-csv), the duration is an integer in **minutes**.

However, consider brief delivery times like just putting a letter in a post box (~45s) or let's say other fractions of minutes like 90 seconds. 

Wouldn't it be better to use either floats (so we can use e.g. 0.5 minutes) or better, just use seconds instead of minutes as unit here?

**Comment by reinterpretcat** [comment]
I think I can easily convert this to seconds: https://github.com/reinterpretcat/vrp/blob/master/vrp-cli/src/extensions/import/csv.rs#L70

**Comment by do-me** [comment]
Perfect! But the duration in the normal problem JSON for VRP is already in seconds right? As far as I checked the docs, I didn't find it explicitly mentioned, maybe you could add it e.g. [here](https://reinterpretcat.github.io/vrp/concepts/pragmatic/problem/jobs.html#places)?

**Comment by reinterpretcat** [comment]
Yes, it is in seconds. I'll add it to the docs too (but they will be published once a new version is released which will take some time)

**Comment by reinterpretcat** [comment]
Changed here: https://github.com/reinterpretcat/vrp/commit/8d3071180f5e1ebbb5d71562fb0ed51072653fd7

so far in master only, will be available with next released version
